src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +64,18 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /** Returns a concatenated version, separated by a separator, of the printable strings of each object **/
+    static String getPrintableString(Printable... printables) {
+        String separator = " ";
+        String resultString = "";
+        int count = 1;
+        for (Printable p : printables) {
+            resultString.concat(p.getPrintableString());
```

Comment:
💡 _Java tip:_ Java 8 has a `StringJoiner` class that can join multiple strings using a delimiter e.g. `aaa` `bbb` `ccc` --> `aaa, bbb, ccc` https://www.mkyong.com/java8/java-8-stringjoiner-example/